parm/config/gefs/yaml/defaults.yaml

```diff
@@ -15,7 +15,20 @@ base:
   FCST_BREAKPOINTS: "48"
   REPLAY_ICS: "NO"
   USE_OCN_PERTURB_FILES: "false"
+  FHOUT_GFS: 6
   FHOUT_OCN_GFS: 6
   FHOUT_ICE_GFS: 6
   HPSSARCH: "NO"
   LOCALARCH: "NO"
+  SFS_POST: "NO"
+fcst:
+  reforecast: "YES"
+  FHZER: 6
+  TYPE: "nh"
+  MONO: "non-mono"
+stage_ic:
+  USE_OCN_ENS_PERTURB_FILES: "YES"
+  USE_ATM_ENS_PERTURB_FILES: "YES"
```

Comment:
Do we really want to default to using these?

Reply:
I changed the default to NO